spec.bs

```diff
@@ -1257,6 +1259,8 @@ with a random salt, making them non-consistent across calls.
 
 # Accessibility Considerations # {#sctn-accessibility-considerations}
 
+User agents render the {{PaymentCredentialInstrument/icon}} and {{PaymentCredentialInstrument/displayName}} together. Relying parties ensure the accessibility of the icon presentation by providing sufficient information via the {{PaymentCredentialInstrument/displayName}}. For example, if the icon represents a bank, include the bank name in the {{PaymentCredentialInstrument/displayName}}.
```

Comment:
Do you intend this sentence to be descriptive for RPs ("Relying parties ensure the accessibility of [...] by [...]"), or should it be imperative ("Relying Parties should ensure ...")?

Reply:
My intention is to be descriptive. (The conformance section aims at software implementations.)

Reply:
Ack, in which case you might want to change 'include the bank name' to 'by including the bank name' ?

Reply:
```suggestion
User agents render the {{PaymentCredentialInstrument/icon}} and {{PaymentCredentialInstrument/displayName}} together. Relying parties ensure the accessibility of the icon presentation by providing sufficient information via the {{PaymentCredentialInstrument/displayName}}. For example, if the icon represents a bank, by including the bank name in the {{PaymentCredentialInstrument/displayName}}.
```